README.md

```diff
@@ -49,11 +49,13 @@ both Bioconductor and tidyverse worlds.
 | `ggplot2`          | `ggplot` (`tidySingleCellExperiment::ggplot`)                                |
 | `plotly`           | `plot_ly` (`tidySingleCellExperiment::plot_ly`)                              |
 
-| Utilities       | Description                                                                       |
-|-----------------|-----------------------------------------------------------------------------------|
-| `tidy`          | Add `tidySingleCellExperiment` invisible layer over a SingleCellExperiment object |
-| `as_tibble`     | Convert cell-wise information to a `tbl_df`                                       |
-| `join_features` | Add feature-wise information, returns a `tbl_df`                                  |
+| Utilities        | Description                                                                       |
+|------------------|-----------------------------------------------------------------------------------|
+| `tidy`           | Add `tidySingleCellExperiment` invisible layer over a SingleCellExperiment object |
+| `as_tibble`      | Convert cell-wise information to a `tbl_df`                                       |
+| `join_features`  | Add feature-wise information, returns a `tbl_df`                                  |
+| `aggregate_cells`| Aggregate expression data within cell groups                                      |
```

Comment:
"Aggregate expression data within cell groups" should read as "Aggregate cell gene-transcription abundance as pseudobulk tissue"

I don't like to use word "expression" when talking about RNA